lib/Common/Common/GetCurrentFrameId.h

```diff
@@ -21,8 +21,13 @@
 #define GET_CURRENT_FRAME_ID(f) \
     __asm { mov f, ebp }
 #elif defined(_M_X64)
+#ifdef _WIN32
 #define GET_CURRENT_FRAME_ID(f) \
     (f = _ReturnAddress())
+#else
+#define GET_CURRENT_FRAME_ID(f) \
+    __asm { mov f, rbp }
```

Comment:
Yup- JC is correct. There is no problem with the implementation of _ReturnAddress. The StackFrame API has two mechanisms with which the stack walker can initialize the frame- either by frame id or with the return address. The Windows AMD64 version of the stack frame initializes basedo n return address. However, we based the Linux stack walker on the Windows x86 one, where the frame id is not the return address but the value of ebp in that stack frame. We could change the stack frame implementation to use the return address instead but the simplest solution for now is to use the frame id. And JC's right, we might have to update this when we work on the JIT.

JC- I'll update to use __builtin_frame_address(0). Thanks for the review, guys!